### PR TITLE
[LOGIC REFINE]refine scout move logic

### PR DESCRIPTION
**Pull request by XenoAmess**

### CODE REVIEW

**Review by mschultheiss83 — APPROVED** [review]
looks good to me

File: src/role_scout.js [hunk 1]
```diff
@@ -17,17 +17,44 @@ roles.scout.settings = {
  * @param {object} creep
```
Comment (mschultheiss83) [inline src/role_scout.js:17]: `* @param {Creep} creep`

Comment (XenoAmess) [reply src/role_scout.js:17]: I would rather do it in another seperate pr, as:
image

Comment (mschultheiss83) [reply src/role_scout.js:17]: sure, fine with that

Comment (mschultheiss83) [reply src/role_scout.js:17]: i'll ping you on that

Comment (XenoAmess) [reply src/role_scout.js:17]: > i'll ping you on that

ping me when your doc fix pr in. or would bring large amout of conflicts.

**Comment by TooAngel** [comment]
This pull request was merged by [worlddriven](https://www.worlddriven.org).

**Pull request merged**